tests/ci/run_ec2_test_framework.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
```

Comment:
suggestion: Probably not consistent elsewhere in our repo, but I tend to like the shebang to be something like `#!/usr/bin/env bash`. Lets the environment control which specific bash command to use by letting `env` search `PATH` for you. It's available in mac as well, and have yet to find a system image that doesn't provide it in a default system. (Likely cause it is part of the GNU coreutils).

Reply:
Sure I just took this from other files in our repo, but it's definately great to have a standard we want to enforce.